cirq-superstaq/cirq_superstaq/daily_integration_test.py

```diff
@@ -136,8 +136,22 @@ def test_get_resource_estimate(service: css.Service) -> None:
 
 def test_get_targets(service: css.Service) -> None:
     result = service.get_targets()
-    assert "ibmq_qasm_simulator" in result["compile-and-run"]
-    assert "aqt_keysight_qpu" in result["compile-only"]
+    ibmq_properties = {
```

Comment:
I was using `ibmq_properties` and `aqt_properties` has helper dictionaries to initialize the following: 

```
 assert gss.typing.TargetInfo(target="ibmq_qasm_simulator", **ibmq_properties) in result
 assert gss.typing.TargetInfo(target="aqt_keysight_qpu", **aqt_properties) in result
```
Did you have an alternative use of `TargetInfo` in mind?

Reply:
i was just thinking it would be slightly cleaner to instantiate them directly instead of using the dict, e.g.
```python
    ibm_target_info = gss.TargetInfo(
        target="ibmq_qasm_simulator",
        supports_submit=True,
        supports_submit_qubo=False,
        supports_compile=True,
        available=True,
        retired=False,
    )
    aqt_target_info = gss.TargetInfo(...)

    assert ibm_target_info in result
    assert aqt_target_info in result
```

Reply:
Sure that's fair, done.